packages/docs-app/src/examples/core-examples/overlayExample.tsx

```diff
@@ -96,8 +96,6 @@ export class OverlayExample extends BaseExample<IOverlayExampleState> {
                 <Switch checked={usePortal} key="portal" onChange={this.handleUsePortalChange}>
                     Use <Code>Portal</Code>
                 </Switch>,
-            ],
-            [
```

Comment:
this is the columns thing i was talking about. no need to separate like this when they're all in one column